# CÃ³digo de error
int32 retcode
# String informativo
string what